app/views/hub/anonymized_intake_csv_extracts/index.html.erb

```diff
@@ -10,21 +10,21 @@
   <table class="data-table document-table">
     <thead>
     <tr>
-      <th class="extract-column__filename">
+      <th scope="col" class="extract-column__filename">
         <%= t(".filename") %>
       </th>
-      <th class="extract-column__record-count">
+      <th scope="col"  class="extract-column__record-count">
```

Comment:
There are two spaces after `"col". ` (could be `"col" `). Not a blocker at all.

Reply:
good catch!